config/schemas/README.md

```diff
@@ -109,3 +109,155 @@ This can be enabled this in other repositories by editing the file `.vscode/sett
   }
 }
 ```
+
+## Guidelines
+
+The uses for the schema is to provide structural and conditional validation here in this repository, and to provide reference documentation in the public documentation repository.
+To make it easier to read, write, and use the schema we have a set of guidelines to make the most out of it that will follow below.
+
+> [!note]
+> The word "schema" here refers to any part of the schema, even subschemas, or schemas in definitions, etc.
+>
+> The example keys given are within the schema, shortened by stripping any `properties` and `items` components.
+
+### Metadata
+
+These guidelines aim to help in creating better quality reference documentation.
+
+#### Titles
+
+- Must be provided, else it becomes rendered as `Untitled`.
+- Should be capitalised, as it becomes part of rendered headers `<title> Schema`.
+- Should be short, as it becomes part of rendered names.
+- Should be based on the key of the schema, as it becomes more intuitive.
+- Should include the context of the schema for common keys, e.g. `Object Storage` and `Object Storage Network Policies`.
+- Should include the context type of the schema last, as it becomes part of rendered headers `<title> <type> Schema`.
+
+#### Descriptions
+
+- Must be [GitHub Flavoured Markdown (GFM)](https://github.github.com/gfm/), as it will be rendered down into markdown documentation.
```

Comment:
Is this true? Isn't this rendered into mkdocs markdown? I think there's a slight difference, or at least has been

Reply:
Yes, but we run enough plugins to not make this matter, so it is easier to just use the same variant across this repository.